modules/quickstart/pages/dfx-deploy-quickstart.adoc

```diff
@@ -0,0 +1,389 @@
++
+[source,bash]
+----
+dfx --help
+----
+
+[[deploy-hello-world]]
+=== Create and deploy Hello, World
+
+After you have downloaded and installed the {sdk-short-name}, you are ready to create and deploy the default sample application.
+
+The default application consists of one `+greet+` function that accepts one text argument and returns the result in a terminal or in a browser.
+
+To create your first application:
+
+[arabic]
+. Open a terminal shell on your local computer, if you don’t already have one open.
+. Create a new project by running the following command:
++
+[source,bash]
+----
+dfx new workflow_hello
+----
++
+The `+dfx new workflow_hello+` command creates a new project directory with template files and a new Git repository for your project.
+. Change to your project directory by running the following command:
++
+[source,bash]
+----
+cd workflow_hello
+----
+. Start the {IC} network locally on your computer by running the following command:
++
+[source,bash]
+----
+dfx start --background
+----
+. Register, build, and deploy all of the canisters specified in the default `+dfx.json+` file by running the following command:
++
+[source,bash]
+----
+dfx deploy
+----
++
+The command output displays information about the operations performed.
+. Call the predefined `+greet+` function in the program by running the following command:
++
+[source,bash]
+----
+dfx canister call workflow_hello greet '("World")'
+----
++
+The command displays the following output for the `+greet+` function:
++
+....
+("Hello, World!")
+....
+. Copy the canister identifier for the `+workflow_hello_assets+` canister and paste it into a browser to test the `+greet+` function front-end.
++
+If you are using the default local host and port, you should have a URL similar to the following:
++
+....
+http://127.0.0.1:8000/?canisterId=cxeji-wacaa-aaaaa-aaaaa-aaaaa-aaaaa-aaaaa-q
+....
+. Copy the canister identifier for the `+workflow_hello+` canister and paste into a browser using the default local host and port to test the `+greet+` function using the Candid web interface.
++
+If you are using the default local host and port, you should have a URL similar to the following:
++
+....
+http://127.0.0.1:8000/candid?canisterId=75hes-oqbaa-aaaaa-aaaaa-aaaaa-aaaaa-aaaaa-q
+....
+
+=== Deploying a second application
+
+The `+Hello, World+` example illustrates the basic workflow using a simple predefined sample application.
+For a more realistic view of the development process, let's build a second application from scratch.
+
+To keep things simple, let's write a  program that is similar to the default sample application but introduces a few new functions and data types.
+
+==== Create a second application
+
+To create a second sample application:
+
+. Open a terminal shell on your local computer, if you don’t already have one open.
+. Create a new project by running the following command:
```

Comment:
Do we need to make it clear that this shouldn't be done in the `workflow_hello` directory created above?  If the reader already has a terminal open and so doesn't open a new one, if following these steps they would still be in the `workflow_hello` project directory.

Reply:
yep, good suggestion.
fixed